test/components/enketo/fill.spec.js

```diff
@@ -29,12 +29,13 @@ describe('EnketoFill', () => {
     });
     button.element.tagName.should.equal('BUTTON');
     button.attributes('aria-disabled').should.equal('true');
-    button.attributes().title.should.equal('Web Form is not available yet. It has not finished being processed. Please refresh later and try again.');
+    button.should.have.ariaDescription('Web Form is not available yet. It has not finished being processed. Please refresh later and try again.');
+    await button.should.have.tooltip();
```

Comment:
It seemed useful to have separate assertions about the `aria-describedby` attribute and about the tooltip even though the two go hand-in-hand.